_data/events.json

```diff
@@ -0,0 +1,235 @@
+         ]
+      },
+      "important_dates": 
+         {
+            "Start of the anonymity period": "2024-02-10",
+            "Submission deadline": "2024-03-10",
+            "Notification of acceptance": "2024-04-24",
+            "Camera ready papers": "2024-04-24",
+            "Workshop": "2024-06-20"
+         }
+   },
+   {
+      "name":"IWSLT 2024",
+      "id":"iwslt-2024",
+      "description":"International Conference on Spoken Language Translation (IWSLT)",
+      "calls_for_papers_deadline":"2024-04-29",
+      "start_date":"2024-08-15",
+      "end_date":"2024-08-16",
+      "location":{
+         "name":"Bangkok, Thailand",
+         "type":"Place",
+         "event_attendance_mode":"MixedEventAttendanceMode"
+      },
+      "organizer":{
+         "type":"Person",
+         "url":"https://iwslt.org/2023/#organizers"
+      },
+      "links": {
+         "iwslt.org/2024/": "https://iwslt.org/2024/"
+      },
+      "important_dates":
+         {
+            "Release of shared task training and developer data":"2024-01-15",
+            "Evaluation":"1 - 15 April",
+            "Paper submission":"2024-04-29",
+            "Notification of acceptance":"2024-06-04",
+            "Camera ready deadline":"2024-06-24",
+            "Pre-recorded video due":"2024-07-22",
+            "Conference":"15 - 16 August"
+         },
+      "speakers":
+         {
+            "Organizers": [
+               "Marine Carpuat, University of Maryland, United States",
+               "Marcello Federico, AWS AI Labs, United States",
+               "Alex Waibel, CMU, United States",
+               "Jan Niehues, KIT, Germany",
+               "Sebastian Stüker, KIT, Germany",
+               "Elizabeth Salesky, JHU, United States",
+               "Atul Kr. Ojha, University of Galway, Ireland"
+            ]
+         },
+      "shared_tasks":[
+         {
+         "name": "Speech-to-speech track",
+         "url": "https://iwslt.org/2024/s2s"
+         },
+         {
+         "name": "Simultaneous track",
+         "url": "https://iwslt.org/2024/simultaneous"
+         },
+         {
+         "name": "Subtitling track",
+         "url": "https://iwslt.org/2024/subtitling"
+         },
+         {
+         "name": "Offline track",
+         "url": "https://iwslt.org/2024/offline"
+         },
+         {
+         "name": "Dubbing track",
+         "url": "https://iwslt.org/2024/dubbing"
+         },
+         {
+         "name": "Low-resource track",
+         "url": "https://iwslt.org/2024/low-resource"
+         },
+         {
+         "name": "Indic track",
+         "url": "https://iwslt.org/2024/indic"
+         }
+   ]
+   },
+   {
+      "name":"NeTTT",
+      "id":"nettt",
+      "description":"New Trends in Translation and Technology",
+      "start_date":"2024-07-03",
+      "end_date":"2024-07-06",
+      "calls_for_papers_deadline":"2024-03-31",
+      "location":{
```

Comment:
I think for location let's just do

```
"location": "Varna, Bulgaria"
```
and 
```
"online": true
```

A mixed event is just an event where both location and online are set to something.

or we can even do

```
"location": "Varna, Bulgaria and online"
```